FDK17プロジェクト/コード/03.サウンド/Lufs.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace FDK
+{
+    /// <summary>
+    /// The Lufs structure is used to carry, and assist with calculations related to,
+    /// Loudness Units relative to Full Scale. LUFS are measured in absolute scale
+    /// and whole values represent one decibel.
+    /// </summary>
+    [Serializable]
+    public struct Lufs
```

Comment:
This new structure is here early for the next chunk of work. I could have left this out and just updated the song db reading and writing code to use dummy booleans and doubles but I figured I might make things a little clearer by including Lufs early.